.github/workflows/docs.yml

```diff
@@ -0,0 +1,34 @@
+name: Rust Docs
+
+on:
+  push:
+    branches: [kiz-developer-hub]
```

Comment:
This job should go away, I only have it here because I push the same code to https://github.com/kianenigma/polkadot-sdk/ deploy. If anyone from CI-CD has a better plan, please let me know.

Reply:
Currently docs are already created in [this job](https://github.com/paritytech/polkadot-sdk/blob/master/.gitlab/pipeline/build.yml#L86-L128) and published in [this job](https://github.com/paritytech/polkadot-sdk/blob/master/.gitlab/pipeline/publish.yml#L4-L64) on each commit to master. I'm afraid that additional github action might create a race condition between jobs that publish docs. I'd suggest to modify/add command in the `build-rustdoc` job.

Reply:
Yes, I think we need to change this line https://github.com/paritytech/polkadot-sdk/blob/495d24d730bb9c2205a5305f383627eabda4a323/.gitlab/pipeline/build.yml#L128

to instead make the default redirect to the developer hub.

Reply:
fyi pushed a change for this